Sources/MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -274,6 +274,16 @@ extension PassiveLocationDataSource {
          - seealso: `WayNameView`
          */
         public static let roadNameKey: NotificationUserInfoKey = .init(rawValue: "roadName")
+        
+        /**
+         A key in the user info dictionary of a `Notification.Name.passiveLocationDataSourceDidUpdate` notification. The corresponding value is a `Measurement<UnitSpeed>` representating the speed of the current road.
```

Comment:
```suggestion
         A key in the user info dictionary of a `Notification.Name.passiveLocationDataSourceDidUpdate` notification. The corresponding value is a `Measurement<UnitSpeed>` representing the maximum speed limit of the current road.
```

For reference:

https://github.com/mapbox/mapbox-navigation-ios/blob/b6175e00dbb3c5c783d5efcabc135e503d034a2b/Sources/MapboxCoreNavigation/RouteLegProgress.swift#L214-L216